Gemfile

```diff
@@ -13,7 +13,7 @@ gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'unicorn'
 gem "therubyracer"
-gem "less-rails", :git => 'git://github.com/suzan2go/less-rails.git', :branch => "fix-nomethoderror-for-sprockets3 "
+gem "less-rails", :git => 'git://github.com/metaskills/less-rails.git'
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)